ui/app/components/pages/confirm-transaction-switch/confirm-transaction-switch.util.js

```diff
@@ -0,0 +1,4 @@
+export function isConfirmDeployContract (txData = {}) {
+  const { txParams = {} } = txData
+  return !txParams.to
```

Comment:
This is consistent with current code, but I wonder if there is a more specific check. It seems likely that we will be handling other transactions in the future that don't have a recipient address. Perhaps we can parse something from `data`?

Reply:
The tools I have for parsing data (abi-decoder and eth-method-registry) don't return anything from the deploy contract data, so I don't think there's a quick fix at our disposal. I asked kumavis about this as well and deploying a contract should be the only case when `txParams.to` is not defined.